src/popup/components/ui/Input.tsx

```diff
@@ -36,20 +36,25 @@ export interface InputProps
   readOnly?: boolean;
   addOn?: React.ReactNode;
   addOnPosition?: 'left' | 'right';
+  addOnRight?: React.ReactNode;
```

Comment:
Instead of having the `addOnPosition`, I would suggest to have diferent namings for the additional content:
* `leadingAddOn`: if it needs to be placed before the `input` component
* `trailingAddOn`: if it needs to be placed after the `input` component